scripts/world_bank/wdi/worldbank.py

```diff
@@ -295,15 +441,14 @@ def output_csv_and_tmcf_by_grouping(worldbank_dataframe, tmcfs_for_stat_vars,
     """
     # Only include a subset of columns in the final csv
     output_csv = worldbank_dataframe[[
-        'StatisticalVariable', 'IndicatorCode', 'ISO3166Alpha3', 'Year', 'Value'
+        'StatisticalVariable', 'IndicatorCode', 'ISO3166Alpha3', 'Year',
+        'Value', 'observationPeriod'
     ]]
 
     # Output tmcf and csv for each unique World Bank grouping.
     df = pd.DataFrame(columns=[
-        'StatisticalVariable',
-        'IndicatorCode',
-        'ISO3166Alpha3',
-        'Year',
+        'StatisticalVariable', 'IndicatorCode', 'ISO3166Alpha3', 'Year',
+        'observationPeriod'
     ])
     with open('output/WorldBank.tmcf', 'w', newline='') as f_out:
```

Comment:
I couldn't locate the code responsible for creating the "output" folder. Is this folder dynamically created during runtime?

Reply:
"output" folder is already present